scripts/cnode-helper-scripts/env

```diff
@@ -30,6 +30,7 @@ CNODE_PORT=6000                                         # Set node port
 #USE_EKG="N"                                            # Use EKG metrics from the node instead of Promethus. Promethus metrics(default) should yield slightly better performance
 #TIMEOUT_LEDGER_STATE=300                               # Timeout in seconds for querying and dumping ledger-state
 #IP_VERSION=4                                           # The IP version to use for push and fetch, valid options: 4 | 6 | mix (Default: 4)
+#PGREST_API="http://127.0.0.1:8050"                     # Uncomment and set to enable PostgREST API queries against DB-Sync
```

Comment:
Wonder if we should add a test via netcat or something to check and unset PGREST_API if the IP:port is not available?

Reply:
sounds like a good idea